graphql/schemabuilder/pagination.go

```diff
@@ -511,21 +530,18 @@ func (c *connectionContext) applyTextFilter(ctx context.Context, nodes []interfa
 	expensiveNodesToKeep := make([]bool, len(nodes))
 	batchedNodesToKeep := make([]bool, len(nodes))
 
+	executors := []*graphql.ExecutionUnit{}
 	if len(filterTextFieldsNotBatched) > 0 {
-		g.Go(func() error {
-			return c.applyTextFilterNotBatched(ctx, nodes, matchStrings, filterTextFieldsNotBatched, nodesToKeep)
-		})
+		executors = append(executors, c.applyTextFilterNotBatched(ctx, nodes, matchStrings, filterTextFieldsNotBatched, nodesToKeep)...)
 	}
 	if len(filterTextFieldsNotBatchedExpensive) > 0 {
-		g.Go(func() error {
-			return c.applyTextFilterNotBatchedExpensive(ctx, nodes, matchStrings, filterTextFieldsNotBatchedExpensive, expensiveNodesToKeep)
-		})
+		executors = append(executors, c.applyTextFilterNotBatchedExpensive(ctx, nodes, matchStrings, filterTextFieldsNotBatchedExpensive, expensiveNodesToKeep)...)
 	}
 	if len(filterTextFieldsBatched) > 0 {
-		g.Go(func() error {
-			return c.applyBatchTextFilter(ctx, nodes, matchStrings, filterTextFieldsBatched, batchedNodesToKeep)
-		})
+		executors = append(executors, c.applyBatchTextFilter(ctx, nodes, matchStrings, filterTextFieldsBatched, batchedNodesToKeep)...)
 	}
+	ctx.Value("pagiantionScheduler").(graphql.WorkScheduler).RunAll(ctx, executors)
```

Comment:
This will panic if the value isn't set or the type conversion fails.  Could you maybe add a comment explaining that this is always set, so it's safe?